src/app/components/wallet-picker/wallet-picker.component.html

```diff
@@ -0,0 +1,21 @@
+<ion-list>
+	<ion-list-header class="font-bold capitalize pl-0" mode="md">
+		{{ title | translate }}
+	</ion-list-header>
+	<ion-item button (click)="onTap(wallet)" *ngFor="let wallet of wallets">
```

Comment:
Same here, we shouldn't be using `press`?

Reply:
(`press`) means long press: https://hammerjs.github.io/recognizer-press/